.github/workflows/test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     name: ${{ matrix.package }}
     needs: prepack
     strategy:
+      fail-fast: false
```

Comment:
This is something we want, but isn't really part of this PR scope.

Reply:
Aye, I dropped it in there for testing on my fork and forgot to delete it 🤦‍♂️